README.md

```diff
@@ -1 +1,132 @@
-# The-Jimmy-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="my website.jpg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>The Jimmy Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```

Comment:
Nice job so far 😎 but as required by the previous reviewer [here](https://github.com/mohamedgamaleldin999999/The-Jimmy-Project/pull/2#discussion_r1176901782) make sure that all the sections here are referenced below, remember only the `FAQ` is optional, 🤷🏽  if you have any doubts or questions about a section, feel free to tag me here or ask around, some sections can be removed such as the `Live Demo` 🚀  which is only applicable once you have a deployed version but some others are necessary. 🤓

Reply:
@didierganthier 
Hello, My friend. I have a question. Is it ok to ask you to manually have a look at my repo before I submit my third submission since it will be the last one. if not, could you please confirm to me if "add javascript" could be considered as valid Future Feature? 
thank you.

Reply:
Hello @mohamedgamaleldin999999 , yes it's perfectly fine to ask me to take a look and yes this would be considered a valid Future Feature 👍🏽

Reply:
Ok. I think I am ready to submit a third submission now. Could you please have a quick look? Thank you.

Reply:
@didierganthier

Reply:
wait

Reply:
You must add it [here](https://github.com/mohamedgamaleldin999999/The-Jimmy-Project/pull/3#issue-1684019391) in thePR description.

Reply:
The README looks good to me 👍🏽

Reply:
Thanks a lot @didierganthier . how about now?

Reply:
Looks great @mohamedgamaleldin999999